tracer/contrib/config.go

```diff
@@ -0,0 +1,40 @@
+package contrib
+
+import "fmt"
+
+var MYSQL_CONFIG = MySQLConfig{
```

Comment:
I'm not sure about these structs. We should'nt really be requiring users to use this. We should just be parsing a DSN. Am I missing something?